yodeploy/virtualenv.py

```diff
@@ -164,6 +151,9 @@ def relocateable_ve(ve_dir):
     log.debug('Making virtualenv relocatable')
     virtualenv.make_environment_relocatable(ve_dir)
 
+    fix_local_symlinks(ve_dir)
```

Comment:
This (and the following line) seems like a shortcoming of virtualenv or a broken aspect of `.make_environment_relocatable`.

I wonder if this would fix it:
https://github.com/pypa/virtualenv/pull/919

oh well.